README.md

```diff
@@ -1,14 +1,21 @@
 # Dash SDK
 
-[![Package Version](https://img.shields.io/github/package-json/v/dashevo/dashjs.svg?&style=flat-square)](https://www.npmjs.org/package/dash)
+[![NPM Version](https://img.shields.io/npm/v/dash)](https://img.shields.io/npm/v/dash)
```

Comment:
Actually shouldn't these badges still link to the NPM repo in case they are clicked?

Reply:
that would be my expectation as a user

Reply:
Yep, that was an accidental change 🙈

Reply:
Fixed